scripts/zones/Upper_Jeuno/npcs/Monberaux.lua

```diff
@@ -86,6 +86,7 @@ function onEventFinish(player,csid,option)
     elseif (csid == 10) then
         player:setVar("PromathiaStatus",0);
         player:addKeyItem(dsp.ki.MYSTERIOUS_AMULET_DRAINED);
+        player:messageSpecial(ID.text.KEYITEM_OBTAINED,dsp.ki.MYSTERIOUS_AMULET);
```

Comment:
Does dsp.ki.MYSTERIOUS_AMULET_DRAINED not display as the same?

Reply:
No idea, but not sure it matters as it's just a message. Can check if needed?

On retail: https://youtu.be/pKxQd3AbPWU?t=253